docs: Add announcement about the standard library process (#1870)
scala/scala-lang: blog/_posts/2026-01-21-standard-library-process.md

---
layout: blog-detail
post-type: blog
by: Tomasz Godzik, VirtusLab
title: Scala Standard Library Process
---

# Contributing to Scala Standard Library

> Starting with Scala 3.8, the Scala standard library is again available for

Reviewer: ```suggestion
After a long freeze, the Scala 3 standard library is again open to
```

contributions. This however means that the main place for contributing is the
Scala 3 repository. Scala 2.13 might benefit from some improvements, but to

> get full access to the new functionality, you need to migrate to Scala 3.

Reviewer: ```suggestion
access most new functionality, you need to migrate to Scala 3.
```
